spring-kafka-docs/src/main/java/org/springframework/kafka/jdocs/started/noboot/Sender.java

```diff
@@ -29,20 +29,20 @@
 // tag::startedNoBootSender[]
 public class Sender {
 
-	public static void main(String[] args) {
-		AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(Config.class);
-		context.getBean(Sender.class).send("test", 42);
-	}
+    public static void main(String[] args) {
+        AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(Config.class);
+        context.getBean(Sender.class).send("test", 42);
+    }
 
-	private final KafkaTemplate<Integer, String> template;
+    private final KafkaTemplate<Integer, String> template;
 
-	public Sender(KafkaTemplate<Integer, String> template) {
-		this.template = template;
-	}
+    public Sender(KafkaTemplate<Integer, String> template) {
+        this.template = template;
+    }
 
-	public void send(String toSend, int key) {
-		this.template.send("topic1", key, toSend);
-	}
+    public void send(String toSend, int key) {
+        this.template.send("topic1", key, toSend);
+    }
```

Comment:
seems we use 4 spaces in lieu of TAB in code snippets in reference. The above code uses TAB instead, ending up with ugly inconsitent indentation in the same code example:
<img width="895" alt="Screenshot 2023-10-29 at 21 14 01" src="https://github.com/spring-projects/spring-kafka/assets/8211458/f7af41ff-ba89-4970-981b-657c60f3ff70">